tests/network-tests/src/flows/membership/updatingProfile.ts

```diff
@@ -35,9 +51,17 @@ export default async function updatingProfile({ api, query }: FlowProps): Promis
       about: 'Updated about',
       avatarUri: 'https://example.com/updated-avatar.jpg',
       externalResources: [
-        { type: MembershipMetadata.ExternalResource.ResourceType.EMAIL, value: 'B@example.com' },
-        { type: MembershipMetadata.ExternalResource.ResourceType.HYPERLINK, value: 'example.com' },
+        {
+          type: MembershipMetadata.ExternalResource.ResourceType.EMAIL,
+          value: 'B@example.com',
+        },
+        {
+          type: MembershipMetadata.ExternalResource.ResourceType.HYPERLINK,
+          value: 'example.com',
+        },
       ],
+      isVerifiedValidator: false,
+      validatorAccount: 'validator address',
```

Comment:
Finally based on the mappings it should be removable:
```suggestion
      validatorAccount: '',
```